lib/environment.rb

```diff
@@ -164,6 +164,17 @@ def admin_console_url
       return @admin_console_url
     end
 
+    def openshift_idp
```

Comment:
I think we can rename this to just `idp`.

Reply:
ok, I will update